### ISSUE TITLE
`ls -R <dir> | jc --ls` leaving off directory

### ISSUE THREAD

**Issue by AndersenJ** [kellyjonbrazil/jc#60]
I have a directory that looks like this:

```
$ ls -R public/comic
public/comic:
00/  01/

public/comic/00:
000.png  002.png  004.png  006.png           007.png           008.png  010.png  012.png           013.png  015.png  017.png  019.png  021.png  023.png  025.png  027.png  029.png
001.png  003.png  005.png  006_zbonus_0.png  007_zbonus_0.png  009.png  011.png  012_zbonus_0.png  014.png  016.png  018.png  020.png  022.png  024.png  026.png  028.png  030.png

public/comic/01:
031_bonus_0.png  031.png  032.png  033.png
```

When I use `ls -R /public/comic | jc --ls` to generate json, it leaves off the `00` directory, but not its contents. It does get the `01` directory. Right now, I'm using `ls -lR /public/comic | jc --ls` as a workaround, which is fine, but I thought it would be good to report this anyways.

**Comment by kellyjonbrazil** [comment]
Hi @AndersenJ - thanks for reporting this! I'll look into it and see what can be done. Unfortunately `ls` without `-l` is one of the more difficult commands to parse because of the ambiguity the output. Usually there is something that can be done, though.

**Comment by kellyjonbrazil** [comment]
@AndersenJ could you let me know what version of jc you are running? (`jc -ap | head`). And what platform/version is this running on?

Also, could you send the files generated by these commands?
```
ls -R public/comic > ls.out
jc ls -R public/comic > ls.json
```
Feel free to anonymize, if necessary. I can use these files to add tests after a potential fix.

Thanks!

**Comment by AndersenJ** [comment]
I've tried on both Ubuntu 18.04 with GNU ls 8.28 and on Arch Linux with GNU ls 8.32. Both have jc 1.10.7. Results appear to be the same on both platforms.

[ls.json.txt](https://github.com/kellyjonbrazil/jc/files/4601334/ls.json.txt)
[ls.out.txt](https://github.com/kellyjonbrazil/jc/files/4601335/ls.out.txt)

**Comment by kellyjonbrazil** [comment]
Ok, I think I found a fix - just need to do some more tests before I can roll it out in the next minor release.

**Comment by kellyjonbrazil** [comment]
This should be fixed in v1.10.10 (`pip3 install --upgrade jc`)
